shell/components/SortableTable/index.vue

```diff
@@ -420,7 +420,7 @@ export default {
         if (old && !neu) {
           this.manualRefreshTimer = setTimeout(() => {
             this.watcherUpdateLiveAndDelayed(neu, old);
-          }, 1000);
+          }, 2000);
```

Comment:
I had to increase this because I cannot quite sync it properly on the workloads (all) list view.

This is something that is very difficult to debug and I thought it only had to do with some kind queueing issue in terms of DOM updates but if this needs to increase then this `setTimeout` might no be the solution to the problem @richard-cox.

A bit without clue on what to do here

Reply:
I think I've fixed this issue @richard-cox 🙌